lib/rspecq/worker.rb

```diff
@@ -120,7 +127,7 @@ def try_publish_queue!(queue)
 
       jobs = jobs.each_with_object({}) do |j, h|
         # heuristic: put untimed jobs in the middle of the queue
-        puts "New/untimed job: #{j}" if timings[j].nil?
+        log_event("New/untimed job: #{j}", "info", job: j) if timings[j].nil?
```

Comment:
nit: I might have bumped this to a `debug` level since it is a valid scenario for new tests.